modules/sdk-coin-atom/src/lib/transferBuilder.ts

```diff
@@ -0,0 +1,13 @@
+import { TransactionBuilder } from './transactionBuilder';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { TransactionType } from '@bitgo/sdk-core';
+
+export class TransferBuilder extends TransactionBuilder {
```

Comment:
Question: Any specific reason why we have a separate `TransferBuilder` class to just do a get `transactionType` or is this how other coins are doing it ?

Reply:
I think when we get further along the line, we will need to decorate the transaction depending on the type we are building (transfer, staking, other)

And yes this is how other coins do this